pattern-library/sass/patterns/_dropdown-menu.scss

```diff
@@ -0,0 +1,173 @@
+// ------------------------------
+// edX Pattern Library: Utilities - Dropdown Menu
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$dropdown-menu-button-font-size: font-size(small) !default;
+$dropdown-menu-list-font-size: font-size(base) !default;
+$dropdown-menu-z-index: 1001 !default;
+$dropdown-menu-button-text-align: left !default;
+$dropdown-menu-list-text-align: left !default;
+$dropdown-menu-width: 170px !default;
+
+$dropdown-menu-focus-border-color: palette(primary, light) !default;
+$dropdown-link-hover-color: palette(primary, light) !default;
+$dropdown-menu-list-hover-color: palette(grayscale, x-dark) !default;
+$dropdown-menu-font-color: $black !default;
+$dropdown-menu-list-font-color: palette(primary, base) !default;
+$dropdown-menu-background-color: $white !default;
+$dropdown-menu-border-color: $gray !default;
+$dropdown-menu-list-border-color: $gray-cool !default;
+$dropdown-menu-list-border-bottom: 1px dotted $dropdown-menu-list-border-color !default;
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+
+
+// ----------------------------
+// #GENERAL
+// ----------------------------
+
+.dropdown-menu-container {
+  height: 40px;
```

Comment:
It seems a little off that this is hard-coded in pixels. However, when I resize the window everything seems to work fine, so maybe that's not an issue. Is ``40px`` right for all browsers including mobile?